src/modules/headset/headset_vrapi.c

```diff
@@ -284,6 +284,7 @@ static bool vrapi_isTouched(Device device, DeviceButton button, bool* touched) {
   switch (button) {
     case BUTTON_TRIGGER: *touched = input->Touches & ovrTouch_IndexTrigger; return true;
     case BUTTON_THUMBSTICK: *touched = input->Touches & ovrTouch_Joystick; return true;
+    case BUTTON_THUMBREST: *touched = input->Touches & ovrTouch_ThumbRest; return true;
```

Comment:
Should this be `ThumbRest` or `LThumbRest and `RThumbRest`?  VrApi docs don't say what the difference is.